### DISCUSSION TITLE
Packages for 0.14 Dev available?

### DISCUSSION THREAD

**Discussion by banthungprong** [blakeblackshear/frigate#9042] [Ask A Question] [answered]
I download packages from the 0.13 Beta from here:
https://github.com/users/blakeblackshear/packages/container/package/frigate

But I do not find 0.14 packages. Do I have to build them myself or am I missing a link?

Thanks in advance

**Comment by blakeblackshear** [comment, marked as answer]
0.14 is still too early for use. It has many bugs and is missing many features.

**Reply by banthungprong** [reply]
Thanks!
